webui/src/default/theme.tsx

```diff
@@ -28,7 +28,7 @@ export default function createDefaultTheme(themeType: 'light' | 'dark'): Theme {
     return createMuiTheme({
         palette: {
             primary: {
-                main: themeType === 'dark' ? '#eeeeee' : '#444',
+                main: themeType === 'dark' ? '#111111' : '#ffffff',
```

Comment:
Nice one-liner! I hope this won't have unintended consequences 😄

Is there a way to maybe make a new Theme just for that one search component? I think if we changed it here we'd have to test the whole Web UI for color changes and side-effects.

Reply:
Thanks for the comment!
Let me see if there's a way to separate the theme for this component. I'm thinking maybe I should create a new theme and apply it to the search icon button individually?

Reply:
I think that would be the better option here indeed. Looping in @amvanbaren, maybe he has some ideas :)

Reply:
I see the main issue has already been fixed. 
There are 2 more things I noticed:
- You don't have to pass `themeType` through props. You can use `const { pageSettings } = useContext(MainContext);` instead. For example: https://github.com/eclipse/openvsx/blob/5df9b741623a7fcee303e8a241067b54becb2110/webui/src/pages/admin-dashboard/namespace-admin.tsx#L21
- You can define style classes instead of using inline styles, e.g. iconButtonLight and iconButtonDark.
https://github.com/eclipse/openvsx/blob/5df9b741623a7fcee303e8a241067b54becb2110/webui/src/pages/extension-list/extension-list-searchfield.tsx#L29-L37
And then you can use it like this:
https://github.com/eclipse/openvsx/blob/5df9b741623a7fcee303e8a241067b54becb2110/webui/src/pages/extension-detail/extension-detail.tsx#L348-L352